src/ResourceManager/Subscription/AzureRM.Subscription.psd1

```diff
@@ -0,0 +1,126 @@
+﻿#
+# Module manifest for module 'PSGet_AzureRM.Subscription'
```

Comment:
@wilcobmsft this module should be called `AzureRM.SubscriptionDefintion` (likewise the project should be called `SubscriptionDefinition`)

Reply:
SubscriptionDefinition is part of the Subscription RP we're adding. We'll likely be adding other resources (e.g. transfers) over time. We'll likely want those to all go into the same module. By naming the module SubscriptionDefinition, I think we're tying this functionality to a specific container within our new RP, rather than tying it to the RP itself.

Reply:
@wilcobmsft is there something else we can call this besides just Subscription? Something along the lines of SubscriptionOperations or SubscriptionConfiguration? We want to avoid confusing the customers by making it seem that this is the module (and SDK) that you use to get and manage subscriptions.

Reply:
@cormacpayne Picking anything but Subscription would be more confusing in my opinion. Most modules follow the naming convention of the RP, which is what we're doing here. Subscription is somewhat unique in that a /subscription predates the RP model. This is part of the reason we're introducing an actual Subscription RP (and agreed upon by ARM): to provide a proper place for subscription-level operations. I could imagine that eventually cmdlets such as Get-AzureRmSubscription in fact become part of the Subscription module, and route their calls to the Subscription RP.

Reply:
CC: @markcowl @twitchax

We should look into fitting these cmdlets into our `AzureRM.Resources` module, which contains our generic cmdlets. We talked about it a little bit and think this is the best spot for your current and future cmdlets. We can sync up early this week and talk about it a little more if there's still any questions.

Reply:
@cormacpayne Let's sync up later today if possible. I believe having a separate module is preferable for a couple of reasons:
1. Our cmdlets will be all about the subscription itself: creation, transfers, renaming, suspending/resuming/etc. In my mind, stuff in AzureRM.Resources should be all about the stuff *inside* a subscription: deployments, resource groups, etc.
2. The underlying spec and SDK is independent as well for the same reason, so keeping things separate in PowerShell seems more consistent.
3. We can update our cmdlets independently.
That said, it's critical for us to ship these cmdlets *somewhere* - we have a few EA partners planning to preview subscription creation on December 8th, and we'd like to make sure they can use our cmdlets by then.

Reply:
@cormacpayne @markcowl As per our discussion offline, we're proceeding with AzureRM.Subscription.Preview as the module name for now. We can close on the final module/module name before GA. I've updated the PR with what I believe are the necessary changes - please look them over and let me know if there's anything I missed.